docker-compose.yml

```
@@ -251,7 +252,11 @@ services:
       postgres:
         condition: 'service_healthy'
     healthcheck:
-      test: [ "CMD-SHELL", "curl -f localhost:8081/healthz" ]
+      test:
+        [
+          "CMD-SHELL",
+          "curl -f localhost:8081/healthz"
+        ]
```

(nit) auto-formatter bug?
```suggestion
      test: [ "CMD-SHELL", "curl -f localhost:8081/healthz" ]
```